### PR TITLE
METRON-1676: Adding date range selector to PCAP filter bar

### PR DESCRIPTION
**Pull request by tiborm**
## Contributor Comments
PR adding the missing date range selector to the PCAP filter bar.

screen shot 2018-07-19 at 6 36 56 pm

## Test script/instructions
- Strat time (From) and end time (To) selectors should appear as the first two elements of the filter bar
- Time selectors should have default values (end time: now, start time: now - 5 days)
- Due to these fields are mandatory, the user should not be able to set time selectors to an invalid state (eg. missing value, null value or any value other than time)

## Pull Request Checklist

Thank you for submitting a contribution to Apache Metron.  
Please refer to our [Development Guidelines](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=61332235) for the complete guide to follow for contributions.  
Please refer also to our [Build Verification Guidelines](https://cwiki.apache.org/confluence/display/METRON/Verifying+Builds?show-miniview) for complete smoke testing guides.  

In order to streamline the review of the contribution we ask you follow these guidelines and ask you to double check the following:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? If not one needs to be created at [Metron Jira](https://issues.apache.org/jira/browse/METRON/?selectedTab=com.atlassian.jira.jira-projects-plugin:summary-panel).
- [x] Does your PR title start with METRON-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [x] Has your PR been rebased against the latest commit within the target branch (typically master)?

### For code changes:
- [x] Have you included steps to reproduce the behavior or problem that is being changed or addressed?
- [x] Have you included steps or a guide to how the change may be verified and tested manually?
- [x] Have you ensured that the full suite of tests and checks have been executed in the root metron folder via:
  ```
  mvn -q clean integration-test install && dev-utilities/build-utils/verify_licenses.sh 
  ```

- [x] Have you written or updated unit tests and or integration tests to verify your changes?
- [x] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [x] Have you verified the basic functionality of the build by building and running locally with Vagrant full-dev environment or the equivalent?

### For documentation related changes:
- [x] Have you ensured that format looks appropriate for the output in which it is rendered by building and verifying the site-book? If not then run the following commands and the verify changes via `site-book/target/site/index.html`:

  ```
  cd site-book
  mvn site
  ```

#### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.
It is also recommended that [travis-ci](https://travis-ci.org) is set up for your personal repository such that your branches are built there before submitting a pull request.

### CODE REVIEW

**Comment by cestella** [comment]
Due to a bug in apache rat, our license checking component, it's ignoring all license checks on typescript files.  I have submitted a PR to up the rat plugin version and have fixed it in master #1126, let's make sure to add license headers here for new typescript files added in this PR though.

**Comment by mmiklavc** [comment]
>  Have you included steps to reproduce the behavior or problem that is being changed or addressed?
> 
>  Have you included steps or a guide to how the change may be verified and tested manually?

Can you submit or link to the test script/instructions for this?

**Comment by justinleet** [comment]
The fields for the date selector are unable to be input manually, which makes doing something simple like changing a year fairly unintuitive.  Is there a way to allow the user to be able to make alterations like this?

**Comment by tiborm** [comment]
@mmiklavc test instructions added to the PR description and the jira ticket as well

**Comment by tiborm** [comment]
@justinleet User able to select a year or month by opening the time selector drop-down and clicking on the year of month values on the top. We reused metron's date-picker component. This behavior should be the same in other parts of the UI.

File: metron-interface/metron-alerts/src/app/pcap/pcap.routing.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/**
+/*
```
Comment (merrimanr) [inline metron-interface/metron-alerts/src/app/pcap/pcap.routing.ts]: Was this intentional?

**Comment by tiborm** [comment]
@merrimanr I fixed the license header typo. It's a copy/paste issue so we'll try to find where is it comes from. Thanks!

**Comment by merrimanr** [comment]
Thanks this looks good to me.  +1

**Comment by merrimanr** [comment]
Can you close this PR?

**Comment by tiborm** [comment]
Thanks Ryan!

**Pull request closed without merging**